src/Listener/ApiListener.php

```diff
@@ -390,7 +391,9 @@ public function viewClass($type, $class = null)
      */
     public function setFlash(Event $event)
     {
-        $event->stopPropagation();
+        if (!$this->config('setFlash')) {
```

Comment:
This is stop the event even when `setFlash` is undefined, which will be the case when the `ApiListener` is not attached and existing apps. Instead explicitly check if value is `false`.